docs/rule/no-invalid-aria-attributes.md

```diff
@@ -0,0 +1,22 @@
+# no-invalid-aria-attributes
+
+This rule checks for the use of invalid ARIA attributes, or any aria-\* property on an element that is not included in the [WAI-ARIA States and Properties spec](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties).
+
+## Examples
+
+This rule **forbids** the following:
+
+```hbs
+<input type="text" aria-require="true" />
```

Comment:
In this example, we want to make it really clear this is not a real attribute, so let's name it something like:
```suggestion
<input type="text" aria-not-real="true" />
```